forge-ai/src/main/java/forge/ai/AiCardMemory.java

```diff
@@ -79,21 +79,21 @@ public enum MemorySet {
     private final Set<Card> memRevealedCards;
 
     public AiCardMemory() {
```

Comment:
@tool4ever @kevlahnota might it be cleaner if we change the structure into this?

`Map<MemorySet, Set<Card>>`, and have the whole logic inside?
Then we don't need extra properties for each MemorySet-Type

Reply:
> @tool4ever @kevlahnota might it be cleaner if we change the structure into this?
> 
> `Map<MemorySet, Set<Card>>`, and have the whole logic inside? Then we don't need extra properties for each MemorySet-Type

already updated the logic